Oneirocom/Magick
# Agent Dropdown Update on agent create

## What Changed:

I added state with will be shared on both components when ever there is any change to agents

## How to test:

Create agent on agent window and check well if newly created agent appears on agent dropdown
this will work when agent is also deleted or renamed

## Additional information:

Any other information that might be useful while reviewing.
